ParisBNF/abb/BNFabb29.xml

```diff
@@ -214,7 +211,6 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                       <msIdentifier>
                 </msIdentifier>
                       <msContents>
-                         <summary/>
```

Comment:
I am always in favour of empty `summary `elements, this will allow the app to print the summary